src/blueapi/cli/cli.py

```diff
@@ -15,60 +14,49 @@
 
 @click.group(invoke_without_command=True)
 @click.version_option(version=__version__, prog_name="blueapi")
+@click.option(
+    "-c",
+    "--config",
+    type=Path,
+    help="Path to configuration YAML file",
+    default=DEFAULT_YAML_PATH,
+)
 @click.pass_context
-def main(ctx) -> None:
+def main(ctx, config: Path) -> None:
     # if no command is supplied, run with the options passed
+    config_loader = ConfigLoader(ApplicationConfig)
+
+    ctx.ensure_object(dict)
+    ctx.obj["config"] = (
+        config_loader.load_from_yaml(config)
+        if DEFAULT_YAML_PATH.exists()
```

Comment:
I am unsure if this will survive packaging, i.e. when you pip install blueapi, I think this may no longer be true.

Reply:
why not? Is it an issue with the global variable `DEFAULT_YAML_PATH`?